tsconfig-es.json

```diff
@@ -0,0 +1,11 @@
+{
+	"extends": "./tsconfig-base.json",
+	"compilerOptions": {
+		"module": "ESNext",
```

Comment:
some language features from esnext are missing in Edge and this is causing syntax errors in that environment. Please use `es6` for both the `module` and `target` properties

Reply:
I got it working on my end by using babel to transpile to something that works on edge too. This is not currently a blocking issue but it would have made things easier.